lib/mix/tasks/ecto.ex

```diff
@@ -0,0 +1,30 @@
+defmodule Mix.Tasks.Ecto do
+  use Mix.Task
+
+  @shortdoc "Prints Ecto help information"
+
+  @moduledoc """
+  Prints Ecto tasks and their information.
+
+  `mix ecto`
```

Comment:
Was wondering whether more docs could go in here to point to more detailed task docs?